logstash-output-s3.gemspec

```diff
@@ -1,13 +1,13 @@
 Gem::Specification.new do |s|
   s.name            = 'logstash-output-s3'
-  s.version         = '4.3.6'
+  s.version         = '4.3.7'
```

Comment:
@robbavey do you think we should change major version? IMO we are fixing the corrupted upload issue so changed minor version. But this change also includes Java-Ruby mixed environment support. Confusing since setup changes does not give any benefits. Need your feedback.